snugget_load.py

```diff
@@ -0,0 +1,205 @@
+import os
+import csv
+
+import psycopg2
+
+def main():
+  appName = "world"
+  appDir = "world"
+  dataDir = os.path.join(appDir, "data")
+  snuggetFile = os.path.join(dataDir, "snuggets.csv")
+  overwriteAll = False
+
+  try:
+    dbURL = os.environ['DATABASE_URL']
+  except:
+    print("Error: DATABASE_URL environment variable is not set. See README.md for how to set it.")
+    exit()
+  
+# dbURL should be in the form protocol://user:password@host:port/databasename  
+  dbParts = [x.split('/') for x in dbURL.split('@')]
+  dbHost = dbParts[1][0].split(":")[0]
+  dbPort = dbParts[1][0].split(":")[1]
+  dbUser = dbParts[0][2].split(":")[0]
+  dbPass = dbParts[0][2].split(":")[1]
+  dbName = dbParts[1][1]
+  
+  with psycopg2.connect(host=dbHost, port=dbPort, user=dbUser, password=dbPass, database=dbName) as conn:
+    with conn.cursor() as cur:      
+      with open(snuggetFile) as csvFile:
+        newSnuggets = csv.DictReader(csvFile)
+        for row in newSnuggets:
+          overwriteAll = processRow(appName, snuggetFile, cur, overwriteAll, row)
+
+
+          
+
+          
+def processRow(appName, snuggetFile, cur, overwriteAll, row):
+  filterColumn = row["shapefile"] + "_filter_id"
+  # "section" -> world_snugget.section_id
+  sectionID = getSectionID(appName, row["section"], cur)
+
+  # check if a snugget for this data already exists
+  # if we have a lookup value then deal with this value specifically:
+  if row["lookup_value"] is not '':  # if it is blank, we'll treat it as matching all existing values
```

Comment:
Consider defining constants for "lookup_value" and "shapefile", if for no other reason than to avoid typos in them later.